### PR TITLE
config: update exclusions for local-checkstyle

### PR DESCRIPTION
**Pull request by nmancus1**
From https://github.com/checkstyle/checkstyle/commit/73ecd6d083778ec7e54b265fab8ec967bcde263b#r62623610

Exclusions for indentation input files are not working, and causing wercker to fail in checkstyle repo on `./.ci/validation.sh no-exception-checkstyle-sevntu`. This could be due to lack of leading double asterisk pattern.

### CODE REVIEW

**Comment by nmancus1** [comment]
Pr to prove this works at https://github.com/checkstyle/checkstyle/pull/11125

**Review by romani — APPROVED** [review]

**Review by rnveach — COMMENTED** [review]
There are more `**/*` missing if this is the issue.

**Comment by romani** [comment]
It is good sign that such excludes are not required

File: checkstyle-tester/projects-for-wercker.properties [hunk 1]
```diff
@@ -5,8 +5,8 @@
...
 
 # 'InputAllEscapedUnicodeCharacters' must be skipped because it is too big and slows down JXR
-# 'InputXpathMapperStringConcat' must be skipped until https://github.com/checkstyle/checkstyle/issues/10810
-#local-checkstyle|local|../../../|master|**/.ci-temp/**/*,**/resources-noncompilable/**/asttreestringprinter/**/*,**/resources-noncompilable/**/filefilters/**/*,**/resources-noncompilable/**/main/**/*,**/resources-noncompilable/**/suppressionsstringprinter/**/*,**/resources-noncompilable/**/gui/**/*,**/resources-noncompilable/**/javadocpropertiesgenerator/**/*,src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/javaparser/InputJavaParser.java,**/InputAllEscapedUnicodeCharacters.java,**/resources-noncompilable/**/javaparser/InputJavaParser.java,**/resources-noncompilable/**/checks/imports/unusedimports/InputUnusedImportsSingleWordPackage.java,src/test/resources/com/puppycrawl/tools/checkstyle/xpath/xpathmapper/**/*,src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/indentation/**/*,**/target/test-classes/**/*
+# 'InputXpathMapperStringConcat' and indentation input files must be skipped until https://github.com/checkstyle/checkstyle/issues/9267
```
Comment (rnveach) [inline checkstyle-tester/projects-for-wercker.properties:8]: @romani 9267 has been merged. This can be reverted right?

Comment (rnveach) [reply checkstyle-tester/projects-for-wercker.properties:8]: Nevermind, I forgot to look at master. It is not there anymore.

**Pull request merged**